src/aad/getMemberDetails.ts

```diff
@@ -36,10 +37,21 @@ export const getMemberDetails = async (): Promise<IMemberDetails[]> => {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
     axios.defaults.headers.common.Authorization = `Bearer ${accessToken}`;
 
-    const response = await axios.get<MemberList>(requestUrl, {
+    let response = await axios.get<MemberList>(requestUrl, {
       headers: { Authorization: `Bearer ${accessToken}`, ConsistencyLevel: 'eventual' },
     });
-    return response.data.value;
+
+    let users = response.data.value
+
+    while (response.data['@nextlink']) {
+      const nextlink = response.data['@nextlink'].replace('accountEnabled+eq+true', 'accountEnabled eq true')
```

Comment:
This pretty ugly line is because the nextlink value that MS sends back from AD converts the spaces to + signs. The plus signs then in turn break the filter (which would mean we'd only ever do the first fetch).

Reply:
Is there a unit test in here that validates that the + are replaced?

Reply:
the unit test added in the last commit verifies it implicitly